.github/README.md

```diff
@@ -3,15 +3,31 @@
   <h1 align="center">Files</h1>
 </p>
 
-[![Build Status](https://dev.azure.com/filescommunity/Files/_apis/build/status/Build%20Pipeline?branchName=main)](https://dev.azure.com/filescommunity/Files/_build/latest?definitionId=4&branchName=main)
-[![Crowdin](https://badges.crowdin.net/files-app/localized.svg)](https://crowdin.com/project/files-app)
-[![Discord](https://discordapp.com/api/guilds/725513575971684472/widget.png)](https://discord.gg/files)
-[![Download](https://img.shields.io/badge/Download%20Installer-blue.svg?style=flat-round)](https://files.community/download)
-[![Documentation](https://img.shields.io/badge/View%20Documentation-purple.svg?style=flat-round)](https://files.community/docs)
+<p align="center">
+  <a style="text-decoration:none" href="https://github.com/files-community/Files/actions/workflows/ci.yml">
+    <img src="https://github.com/files-community/Files/actions/workflows/ci.yml/badge.svg" alt="Files CI Status" /></a>
+  <a style="text-decoration:none" href="https://crowdin.com/project/files-app">
+    <img src="https://badges.crowdin.net/files-app/localized.svg" alt="Files Localization Status" /></a>
+  <a style="text-decoration:none" href="https://discord.gg/files">
+    <img src="https://dcbadge.vercel.app/api/server/files?style=flat" alt="Files Discord" /></a>
+  <a style="text-decoration:none" href="https://files.community/docs">
+    <img src="https://img.shields.io/badge/View%20Documentation-purple.svg?style=flat-round" alt="Documentation link" /></a>
+</p>
+
+Files is a modern file manager that helps users organize their files and folders. Our mission with Files is to build the best file manager for Windows, and we’re proud to be building it out in the open so everyone can participate. User feedback helps shape the features we work on, & the bug reports on GitHub help to make Files more reliable. Built and maintained by the open-source community, Files features robust multitasking experiences, file tags, deep integrations, and an intuitive design.
+
+## Install Files
+
+### Via Microsoft Store
+
```

Comment:
Should we add the description from the download page?

"Purchasing Files through the Microsoft Store helps support the developers and allows us to continue updating the app with new features and improvements."

Reply:
Yeah, important to denote *purchasing*.